### DISCUSSION TITLE
Broken GUI for high DPI configuration (Windows)

### DISCUSSION THREAD

**Discussion by lgmacri** [DISTRHO/Cardinal#120] [General]
Hi! 

I just wanted to give this a try, I think it's a great idea. 

However I could not make it work so far (Windows x64, latest FL Studio x64)

This is what I see:

<img width="777" alt="Cardinal" src="https://user-images.githubusercontent.com/98064202/150440508-5dcffda7-d6a6-4bbb-9b5c-11aefb9c52a7.PNG">

I just downloaded the latest builds (both Win32 and Win64) and copied the complete folder CardinalSynth.vst to my VST2 directory and CardinalSynth.vst3 to my VST3 directory and I get the same result.

Also trying to interact with the GUI causes the plugin to crash.

**Comment by falkTX** [comment]
You either dont have the graphics drivers installed for your machine, or your machine does not support OpenGL2 which is required for showing the GUI.

**Reply by lgmacri** [reply]
I just updated my graphics driver just in case (GeForce RTX 2060, driver version 511.23) and still got the same result. 

However I found something interesting: I'm using a high DPI monitor with Windows scaling set to 150%. If I set the scaling to 100% then the plugin loads fine!

**Reply by MiguelRita** [reply]
Can confirm that scaling to 100% works fine, but not ideal.
